smithy-openapi/src/main/java/software/amazon/smithy/openapi/fromsmithy/OpenApiJsonSchemaMapper.java

```diff
@@ -106,6 +106,22 @@
         return builder;
     }
 
+    private void updateFloatFormat(Schema.Builder builder, JsonSchemaConfig config, String format) {
```

Comment:
This is a bit unfortunate. I wonder if later we can rethink the divide between OpenAPI and JSON Schema converters.